starboard/build/config/BUILDCONFIG.gn

```diff
@@ -385,13 +454,20 @@ template("shared_library") {
     install_target(target_name + "_install") {
       forward_variables_from(invoker, [ "testonly" ])
       installable_target_name = shared_library_target_name
+      installable_target_dep = ":$original_target_name"
```

Comment:
this is a bug, needs to be installable_target_dep = "$original_target_name"
remove the ":"

Reply:
Talked to Andrew.
The idea is to pass deps as it is via the invoker So the changes have to made on the ps4 side to absorb it.

Currently ps4 creates deps based on name https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:internal/starboard/ps4/install_target.gni;l=27?q=p:cobalt%20ps4%2Finstall_target.gni

that has to change to absorb deps directly.